content/reference-rippled.md

```diff
@@ -2199,8 +2197,9 @@ The request includes the following parameters:
 | `limit`            | Integer                                    | (Optional, default varies) Limit the number of transactions to retrieve. The server is not required to honor this value. |
 | `marker`           | [(Not Specified)](#markers-and-pagination) | Value from a previous paginated response. Resume retrieving data where that response left off. This value is stable even if there is a change in the server's range of available ledgers. |
 
-[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/rpc/handlers/AccountTxSwitch.cpp "Source")
-There is also a deprecated legacy variation of the `account_tx` method. For that reason, we recommend *not using any of the following fields*: `offset`, `count`, `descending`, `ledger_max`, `ledger_min`.
+[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/rpc/handlers/AccountTxSwitch.cpp "Source")<br>
+
+**Note:** There is also a deprecated legacy variation of the `account_tx` method. For that reason, we recommend *not using any of the following fields*: `offset`, `count`, `descending`, `ledger_max`, and `ledger_min`. If you include any of these deprecated fields in the request, the method will not support pagination.
```

Comment:
Nitpick: avoid future tense here. Suggest you change "will not support" to "does not support".

Reply:
I appreciate all nitpicks. Crucial to keeping things in order! Fixed. I saw it as I was committing and should have fixed it. Embarassed 💯 . Hehe. Thank you...